### DISCUSSION TITLE
Update rAudio?

### DISCUSSION THREAD

**Discussion by jtritter** [rern/rAudio#1208] [General]
Hi there,

maybe I am just to stupid to find out by myself, but how can I update my rAudio-System? I installed with an images of release 20221127 and I can see here on github that there is a new release 20230123. But I have no option in the WEB UI to update. I'm using an Raspberry Pi 3B. Do I have to create a new image for this reason? And do all the configuration stuff once again? By the way: I have the whole music on the same sd card - so I would need to copy all files again :-( Is this the way for every update? Or is there a way via command line?

Or am I just to blind to see the right button for updates?

Yours
Jürgen

P.S.: rAudio is absolutly great - the best Audio player I ever had on my RasPi (and I tried a lot of stuff)...

**Comment by cYCL157** [comment]
If you don't see an option to update in settings -> addons -> update -- then I would think that that image was before the last forced new image update awhile ago.  You might need to backup your settings and then flash new sd and restore your settings.

**Comment by jtritter** [comment]
Thank you for the advice. There is an information about an update in "Settings --> Addons --> Update" but no option to load/install this update :-( .
![image](https://user-images.githubusercontent.com/76902662/214784537-cb5bfb17-3f3f-414b-8a2b-0f1ed566b5ae.png)
The images of release 20221117 was downloaded and flashed/installed last Saturday (2023-01-21). Strange thing....

I will try your solution. Thanks.

**Comment by ceejayemm** [comment]
You need to flash a new SD card with the latest image version (i20220122) or edit this file /srv/http/data/addons/r1  This file simply contains a date in YYYYMMDD format.  Change the date to (say) 20221215, save the change and try the Update via Addons again.

Chris

**Comment by jtritter** [comment]
The hack with rewriting the  /srv/http/data/addons/r1 worked! But only worked with 20221215! No other day...
Thanks a lot.

**Comment by HunLucas** [comment]
Is Spotify working on the newest update for you?

**Reply by jtritter** [reply]
Sorry, I don't have Spotify, so I can't tell you something about this topic. I'm only using rAudio for local music files.
